mypy.ini

```diff
@@ -0,0 +1,29 @@
+[mypy]
+python_version = 2.7
+
+ignore_missing_imports = True
+
+show_error_codes = True
+
+follow_imports = silent
```

Comment:
Add todo here to change this after we type all files.

Reply:
Added.